test/Microsoft.ML.TimeSeries.Tests/TimeSeries.cs

```diff
@@ -124,6 +124,8 @@ public void SavePipeSlidingWindowW1L1()
         }
 
         [Fact]
+        //Skipping test temporarily. This test will be re-enabled once the cause of failures has been determined
+        [Trait("Category", "SkipInCI")]
         public void SavePipeSlidingWindowW2L1()
         {
             TestCore(null, true,
```

Comment:
I assume folks already know (and are working from it) that Azure DevOps gives us a list of the tests that filed in the last 30 days -- https://dev.azure.com/dnceng/public/_test/analytics?definitionId=104&contextType=build

![image](https://user-images.githubusercontent.com/4080826/73345169-6d71d580-4238-11ea-89e1-ffe6f2f58b7a.png)

All of those are either disabled in the last PR or this one, besides: `Microsoft.ML.RunTests.TestEntryPoints.EntryPointSDCAMulticlass`.

Reply:
Yes, we are aware of this, thanks. This list itself is not accurate as this list will skip any test that failed but success after retry. 

---
In reply to: [372291720](https://github.com/dotnet/machinelearning/pull/4726#discussion_r372291720) [](ancestors = 372291720)